package.json

```diff
@@ -31,8 +31,8 @@
     "url": "https://github.com/aurora-is-near/rainbow-bridge-client"
   },
   "engines": {
-    "node": "15.3.0",
-    "npm": "7.5.4"
+    "node": "14.17.0",
+    "npm": "6.14.13"
```

Comment:
Why downgrading

Reply:
I think it was due to a build issue with the webapp. And v15 is an odd version so it is not maintained. We can make it ">=14.17.0".
https://nodejs.org/en/about/releases/